airbyte-integrations/connectors/source-paypal-transaction/source_paypal_transaction/manifest.yaml

```diff
@@ -150,63 +163,238 @@ definitions:
+  #Paypal API only has V1 for this stream
+  list_disputes_stream:
+    type: DeclarativeStream
+    primary_key: dispute_id
+    name: "list_disputes"
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        $ref: "#/definitions/selector"
+      paginator:
+        type: DefaultPaginator
+        page_token_option:
+          type: RequestOption
+          inject_into: request_parameter
+          field_name: next_page_token
+        page_size_option:
+          type: RequestOption
+          inject_into: request_parameter
+          field_name: page_size
+        pagination_strategy:
+          type: PageIncrement
+          start_from_page: 1
+          page_size: 50
+      requester:
+        $ref: "#/definitions/requester"
+        http_method: GET
+    transformations:
+      - type: AddFields
+        fields:
+          - path:
+              - updated_time_cut
+            value: >-
+              {{ record['update_time'] }}
+    incremental_sync:
+      type: DatetimeBasedCursor
+      cursor_field: updated_time_cut
+      datetime_format: "%Y-%m-%dT%H:%M:%S.%fZ"
+      start_datetime:
+        type: MinMaxDatetime
+        datetime: "{{ format_datetime(config['dispute_start_date'] if config['dispute_start_date'] else (now_utc() - duration('P179D')), '%Y-%m-%dT%H:%M:%S.%fZ')[:23] + 'Z' }}"
+        datetime_format: "%Y-%m-%dT%H:%M:%S.%fZ"
+      end_datetime:
+        type: MinMaxDatetime
+        #Adding a time delta as the API has a problem with the slice being too close to the now_utc. Set to 30M
+        datetime: >-
+          {{ format_datetime(config['dispute_end_date'] if config['dispute_end_date'] else (now_utc() - duration('PT30M')), '%Y-%m-%dT%H:%M:%S.%fZ')[:23] + 'Z'}}
+        datetime_format: "%Y-%m-%dT%H:%M:%S.%fZ"
+      start_time_option:
+        type: RequestOption
+        field_name: update_time_after
+        inject_into: request_parameter
+      end_time_option:
+        type: RequestOption
+        field_name: update_time_before
+        inject_into: request_parameter
+      step: "P{{ config.get('time_window', 7) }}D"
+      cursor_granularity: PT1S
+    $parameters:
+      path: "v1/customer/disputes"
+      field_path: items
+
+  #Stream Search Invoices
+  # Currently it does not support incremental sync as metadata does not contain last_update_date
+  search_invoices_stream:
+    type: DeclarativeStream
+    primary_key: id
+    name: "search_invoices"
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        $ref: "#/definitions/selector"
+      paginator:
+        type: DefaultPaginator
+        page_token_option:
+          type: RequestOption
+          inject_into: request_parameter
+          field_name: page
+        page_size_option:
+          inject_into: request_parameter
+          field_name: page_size
+          type: RequestOption
+        pagination_strategy:
+          type: PageIncrement
+          start_from_page: 1
+          page_size: 100
```

Comment:
is there a different limit for different endpoints? I've seen 20, 50, and 100

Reply:
Yes. Unfortunately Paypal APIs are all over the place, there is no standard between values and this is why all have different pagination values.